content/blog/move-resources-between-stacks/index.md

```diff
@@ -82,13 +82,11 @@ Flags:
+Both `dest` and `source` can be either stacks in the current project, or stacks in a different project, using the fully qualified stack names.  Note that this works only for stacks within the same backend, it is currently not possible to move a resource between different backends (though you can [move stacks between backends](/docs/concepts/state/#migrating-between-state-backends) using other existing tools).
 
-The resources being moved have to be specified by their full [URN](/docs/concepts/resources/names/#urns), and multiple URNs can be passed at once.  For each resource being moved, all the children of that resource will also be moved, and the relationships between all resources being moved is preserved.  Resources with other types of dependencies will however not be moved to the target stack by default.  The easiest way to get the full URN of the resources is to use `pulumi stack --show-urns`.  Note that URNs can contain characters that get interpreted by the shell, so it is always a good idea to wrap them in single quotes when passing them as arguments.
+The resources being moved have to be specified by their full [URN](/docs/concepts/resources/names/#urns), and multiple URNs can be passed at once.  For each resource being moved, all the children of that resource will also be moved, and the relationships between all resources being moved is preserved.  Resources with other types of dependencies will however not be moved to the target stack by default.  The easiest way to get the full URN of the resources is to use `pulumi stack --show-urns`.  Note that URNs can contain characters that get interpreted by the shell, so it is always a good idea to wrap them in single quotes (`'`) when passing them as arguments.
 
-The command will make no attempt to adjust the users program.  Inputs and outputs from broken dependencies will have to be adjusted in the program where the resource is being managed after being moved.  This can be done either by using [stack references](https://www.pulumi.com/docs/concepts/stack/#stackreferences), or recreating the inputs in the program.
-
-The `pulumi state move` command is available in Pulumi v3.126.0 and later.
+The `pulumi state move` command won't modify the code of your program directly, it will only modify the state of the source and desitnation stacks.  So following runnint the command, you should ensure that you modify both programs to match the changes you have made.  This can typically be accomplished by copy/pasting source code for the resources and/or components between the two codebases. Inputs and outputs of resources that were moved may need to be adjusted as part of this process. This can be done either by using [stack references](/docs/concepts/stack/#stackreferences), or recreating the inputs in the program.
```

Comment:
```suggestion
The `pulumi state move` command won't modify the code of your program directly, it will only modify the state of the source and destination stacks.  So following running the command, you should ensure that you modify both programs to match the changes you have made.  This can typically be accomplished by copy/pasting source code for the resources and/or components between the two codebases. Inputs and outputs of resources that were moved may need to be adjusted as part of this process. This can be done either by using [stack references](/docs/concepts/stack/#stackreferences), or recreating the inputs in the program.
```